Spec/RestClient.swift

```diff
@@ -173,8 +173,20 @@ class RestClient: QuickSpec {
                     
                     expect(testHTTPExecutor.requests.first?.URL?.host).toEventually(equal("fake.ably.io"), timeout: testTimeout)
                 }
+
+                it("should ignore an environment when host is not in production") {
```

Comment:
Sorry, that's not what's happening here at all. Well, da76de0cdf5baa0dc4ebae5a3ba41b92deffce82 it is.